arch/mx.rst

```diff
@@ -1,13 +1,10 @@
 .. _mx:
 
-Masterless Mode (beta)
-======================
+Masterless Mode
+===============
 
 Citus MX is a new version of Citus that adds the ability to use hash-distributed tables from any node in a Citus cluster, which allows you to scale out your query throughput by opening many connections across all the nodes. This is particularly useful for performing small reads and writes at a very high rate in a way that scales horizontally. Citus MX is currently available in private beta on `Citus Cloud <https://www.citusdata.com/product/cloud>`_.
 
-Architecture
```

Comment:
Any reason to remove this?

Reply:
After the file move (`cloud/additional_mx.rst` → `arch/mx.rst`), the MX section now lives under Architecture in the TOC.

"Architecture > Citus MX > Architecture" looked kind of strange when expanding the TOC.